android/src/main/kotlin/com/doordeck_flutter_plugin/DoordeckFlutterPlugin.kt

```diff
@@ -27,12 +31,23 @@ class DoordeckFlutterPlugin : FlutterPlugin, MethodCallHandler {
         context = flutterPluginBinding.applicationContext
     }
 
+    override fun onDetachedFromActivityForConfigChanges() = Unit // NO-OP
+
+    override fun onReattachedToActivityForConfigChanges(binding: ActivityPluginBinding) = Unit // NO-OP
+
+    override fun onDetachedFromActivity() = Unit // NO-OP
+
+    override fun onAttachedToActivity(binding: ActivityPluginBinding) {
+        activity = binding.activity
+    }
+
     override fun onMethodCall(call: MethodCall, result: Result) {
         when (call.method) {
             SHOW_UNLOCK_CALL -> showUnlock(result)
             UPDATE_TOKEN_CALL -> updateToken(call, result)
             LOGOUT_CALL -> logout(result)
             INIT_CALL -> initDoordeck(call, result)
+            UNLOCK_WITH_UUID_CALL -> unlockWithUUID(call, result)
```

Comment:
Yeah i definitely prefer device ID :)

Reply:
Can we at least keep this as the method signature on Android is released already? at least to keep it consistent